### ISSUE TITLE
Rename Login field: "Server" -> "Server URL"

### ISSUE THREAD

**Issue by QJKX** [dkanada/gelli#126]
Thanks for the project.

The login screen has a field for "server". The field actually expects a URL (and didn't work for me without the protocol prefix. This is different to the Android Jellyfin client).

I think it would be clearer if the "Server" field was renamed "Server URL".

**Comment by poopybunghole** [comment]
Agreed.  I kept typing the server IP and a domain name with no result.  Eventually I tried http:// and it worked.  For the longest time I thought the app was simply broken.  Maybe have a selectable toggle for http:// or https:// or have a pre filled out url like http://myjellyfin.server so people know what to do without guessing.

**Comment by Korvox** [comment]
The right solution from a UX perspective would probably be to try https:// and then http:// for any entered string to see if its valid. Its just how most url input forms work even if its a bit of a hassle.

**Comment by Korvox** [comment]
The prefill sounds like a great UX solution. Maybe just have http:// in there prefilled, people will know what to do with the rest, so when you edit over the placeholder you don't have to type that out. Only concern would be if it does proper upgrading against hsts (preload or response header) for https only sites.

**Comment by m00nwtchr** [comment]
For a placeholder `https` would be better,  since there's no reason to assume the server doesn't support it in this day and age, and `http` might make people think the app only supports http
